rfc_backmatter.md

```diff
@@ -50,3 +50,575 @@ Taking the last example where 31 bits were needed for the prediction, the requir
+00000054: 6246 4c41 4320 312e 332e 3320  bFLAC 1.3.3
+00000060: 3230 3139 3038 3034 0100 0000  20190804....
+0000006c: 0e00 0000 5449 544c 453d d7a9  ....TITLE=..
+00000078: d79c d795 d79d 8100 0006 0000  ............
+00000084: 0000 0000 fff8 6998 000f 9912  ......i.....
+00000090: 0867 0162 3d14 4299 8f5d f70d  .g.b=.B..]..
+0000009c: 6fe0 0c17 caeb 2100 0ee7 a77a  o.....!....z
+000000a8: 24a1 590c 1217 b603 097b 784f  $.Y......{xO
+000000b4: aa9a 33d2 85e0 70ad 5b1b 4851  ..3...p.[.HQ
+000000c0: b401 0d99 d2cd 1a68 f1e6 b810  .......h....
+000000cc: fff8 6918 0102 a402 c382 c40b  ..i.........
+000000d8: c14a 03ee 48dd 03b6 7c13 30    .J..H...|.0
+```
+
+### Example file 2 in binary representation (only audio frames)
+
+```
+00000088: 11111111 11111000 01101001 10011000  ..i.
+0000008c: 00000000 00001111 10011001 00010010  ....
+00000090: 00001000 01100111 00000001 01100010  .g.b
+00000094: 00111101 00010100 01000010 10011001  =.B.
+00000098: 10001111 01011101 11110111 00001101  .]..
+0000009c: 01101111 11100000 00001100 00010111  o...
+000000a0: 11001010 11101011 00100001 00000000  ..!.
+000000a4: 00001110 11100111 10100111 01111010  ...z
+000000a8: 00100100 10100001 01011001 00001100  $.Y.
+000000ac: 00010010 00010111 10110110 00000011  ....
+000000b0: 00001001 01111011 01111000 01001111  .{xO
+000000b4: 10101010 10011010 00110011 11010010  ..3.
+000000b8: 10000101 11100000 01110000 10101101  ..p.
+000000bc: 01011011 00011011 01001000 01010001  [.HQ
+000000c0: 10110100 00000001 00001101 10011001  ....
+000000c4: 11010010 11001101 00011010 01101000  ...h
+000000c8: 11110001 11100110 10111000 00010000  ....
+000000cc: 11111111 11111000 01101001 00011000  ..i.
+000000d0: 00000001 00000010 10100100 00000010  ....
+000000d4: 11000011 10000010 11000100 00001011  ....
+000000d8: 11000001 01001010 00000011 11101110  .J..
+000000dc: 01001000 11011101 00000011 10110110  H...
+000000e0: 01111100 00010011 00110000           |.0
+```
+
+### Signature and streaminfo
+
+Most of the streaminfo block is the same as in example 1, so only parts that are different are listed in the following table
+
+Start  | Length  | Contents           | Description
+:------|:--------|:-------------------|:-----------------
+0x04+0 | 1 bit   | 0b0                | Not the last metadata block
+0x08+0 | 2 byte  | 0x0010             | Min. blocksize 16
+0x0a+0 | 2 byte  | 0x0010             | Max. blocksize 16
+0x0c+0 | 3 byte  | 0x000017           | Min. frame size 23 byte
+0x0f+0 | 3 byte  | 0x000044           | Max. frame size 68 byte
+0x15+4 | 36 bit  | 0b0000, 0x00000013 | Total no. of samples 19
+0x1a   | 16 byte | (...)              | MD5 signature
+
+This time, the minimum and maximum blocksizes are reflected in the file: there is one block of 16 samples, but the last block (which has 3 samples) is excluded from this number. The MD5 signature is d5b0564975e98b8d8b930422757b8103, this will be verified at the end of this example.
```

Comment:
```suggestion
This time, the minimum and maximum blocksizes are reflected in the file: there is one block of 16 samples, but the last block (which has 3 samples) is excluded from this number. The MD5 signature is 0xd5b0 5649 75e9 8b8d 8b93 0422 757b 8103, this will be verified at the end of this example.
```